src/ui/server/helpers/task-list/index.ts

```diff
@@ -1,6 +1,25 @@
 import { taskStatus, taskLink } from './task-helpers';
 import { TaskListData, TaskListDataTask, TaskListGroup, ApplicationFlat } from '../../../types';
 
+/**
+ * mapTask
+ * @param {TaskListDataTask} Task list groups and tasks
+ * @param {ApplicationFlat} Submitted application data
+ * @returns {Object} Tasks with status and optional href
+ */
+export const mapTask = (task: TaskListDataTask, submittedData: ApplicationFlat) => {
+  const mapped = {
+    ...task,
+    status: taskStatus(task, submittedData),
+  };
+
+  if (task.href) {
```

Comment:
Should this be `task.href` since `src/ui/types/task-list.ts` sets `href` as optional?

Reply:
Yes, it's checking if a href has been provided (because it's optional). If so, it will add a href to the task.